### DISCUSSION TITLE
Huge performance difference !

### DISCUSSION THREAD

**Discussion by pyjacques** [MrNeRF/LichtFeld-Studio#47] [Q&A] [answered]
![image](https://github.com/MrNeRF/gaussian-splatting-cuda/assets/44160710/20e9fc49-ce34-4bc2-9226-2f1660704ec2)
I'm using windows version under pinokio env (https://github.com/bycloud-AI/gaussian-splatting-Windows)
I've done a bench with the truck images used in your benchmark.
Your result is about 3-4 minutes and mine : almost 15minutes ! Why ?
Any suggestion ? Worth try your folk under WLS2/Unbuntu ? 
Win 11 Insider : Ryzen 9 5900x + RTX 3090 FE

**Comment by MrNeRF** [comment, marked as answer]
Hey, I really don't know what the issue is but the original implementation should be around 40% slower than mine. So your far beyond what should be possible. Yeah, maybe try WSL but I am not very familiar with windows and WSL. Seems to be something else that slows is down. Your specs are pretty up-to-date.

**Reply by pyjacques** [reply]
Thanks. Indeed, this is not an issue. Just want to know if i was doing something wrong. I'll try your implementation WSL2, then in dual boot with ubuntu. Nice work BTW ! Have a good day. 😉
